stock_barcodes/templates/missing_moves_template.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <template id="missing_moves">
+        <div class="table-responsive">
+        <table class='o_list_view table table-sm table-hover table-striped o_list_view_ungrouped'>
+            <thead>
+            <tr>
+                <th class="o_column_sortable">Product</th>
+                <th class="o_column_sortable">Initial Demand</th>
+                <th class="o_column_sortable">Done</th>
+            </tr>
+            </thead>
+            <tr t-foreach="moves" t-as="move">
+                <td>
+                    <span t-field="move.product_id"/>
+                </td>
+                <td>
+                    <span t-field="move.product_uom_qty"/>
+                </td>
+                <td>
+                    <span t-field="move.quantity_done"/>
+                </td>
+            </tr>
+            <tfoot class="oe_foot">
+                <tr>
+                    <td></td>
+                    <td></td>
+                    <td></td>
```

Comment:
Do we need this empty elements?

Reply:
I did it to simulate the odoo table as much as possible. If I am not wrong, using the web inspector I saw that odoo uses this method to simulate the gray line at the bottom of the table.

Reply:
I am not able to open runbot somehow. Can you please try to remove the empty elements and check if that works ? @pedrobaeza @sergio-teruel Can you please share your inputs here ?

Reply:
On Odoo, it is done in a similar way, but on js, as you can see here:

https://github.com/odoo/odoo/blob/12.0/addons/web/static/src/js/views/list/list_renderer.js#L394

Reply:
You can use <td colspan="3"/>

Reply:
:+1: